src/routes/ExercisesScreens.tsx

```diff
@@ -94,10 +94,14 @@ const ExercisesScreen = ({ route, navigation }: ExercisesScreenPropsType): JSX.E
   }
 
   const handleNavigation = (item: ExerciseType): void => {
-    setSelectedKey(item.key.toString())
-    navigation.navigate(EXERCISES[item.key].nextScreen, {
-      discipline: discipline
-    })
+    if (item.title === 'Level 1' && numberOfChildren < 4) {
```

Comment:
Please check if the `item.title` equals the constant from the labels.
(import labels-file and check for labels.exercises.wordChoice.title)